lonestar/analytics/cpu/k-core/kcore_cli.cpp

```diff
@@ -107,7 +108,9 @@ main(int argc, char** argv) {
   }
 
   tsuba::TxnContext txn_ctx;
-  if (auto r = KCore(pg.get(), kCoreNumber, "node-in-core", &txn_ctx, plan);
+  if (auto r = KCore(
+          pg.get(), kCoreNumber, "node-in-core", &txn_ctx, symmetricGraph,
+          plan);
```

Comment:
isn't it better to pass the extra arguments like symmetricGraph through "plan"?

Reply:
Not really. Plan should only have things which do not change the output. Symmetric flag will change the output if used incorrectly. Also, very soon we will push this functionality to the RDG itself so that we can auto check if graph is symmetric or not.

Reply:
I mean, it seems that "plan" is controlling how the app behaves and be run. So we can keep all the configs in "plan". does that make sense?

Reply:
> 

So in this case; it makes sense to me. thanks for the clarification.

Reply:
> Not really. Plan should only have things which do not change the output. Symmetric flag will change the output if used incorrectly. Also, very soon we will push this functionality to the RDG itself so that we can auto check if graph is symmetric or not.

So in this case; it makes sense to me. thanks for the clarification.